### ISSUE TITLE
Steamplay/Proton not updating while Boxtron is installed

### ISSUE THREAD

**Issue by samualblair** [dreamer/boxtron#34]
Not sure if this requires a change in boxtron to resolve, or if this issue just needs a documentation update to warn/notify users.

But I have found on several systems over the last 6-8 months that new versions of Steamplay/Proton do no appear in Steam compatibility section while Boxtron is installed.
I use Boxtron installed for a single user on Ubuntu (various versions, recently 19.10).

As a workaround I have found that simply removing Boxtron from the compatability folder while steam is closed, then opening steam is enough to allow Steamplay/Proton to update. Example:
'mv ~/.steam/root/compatibilitytools.d/boxtron ~/'

Then once updated I can re-add Boxtron and it continues to work fine. Example (after above):
'mv ~/boxtron ~/.steam/root/compatibilitytools.d/'

**Comment by dreamer** [comment]
Are you talking about "Steam Play" section in Steam's "Settings" window or "Force the use of a specific Steam Play compatibility tool" in a game "Properties"?

If you're talking about "Settings" - do you have "Enable Steam Play for all other titles" checkbox selected?

**Comment by samualblair** [comment]
Yes that is what I am talking about. It still allows me to use the versions of "Steam Play" proton that are installed prior to installing Boxtron. But no new versions will download or be selectable in that setting, that is until removing Boxtron. Once Boxtron is removed steam recognizes that new versions have been released, will download them automatically, and they are in the selection. After this I can reinstall (move back) Boxtron and continue to use the new versions.

So in summary when Boxtron is installed the auto-update feature of "Steam Play" appears to be broken or disabled.

**Comment by samualblair** [comment]
I should also clarify, when I am seeing this problem it is manifest in the all games section "Enable Steam Play for all other titles" as well as an individual game setting  ""Force the use of a specific Steam Play compatibility tool" options. This is why my thoughts are the system as a whole is not breaking or disabling "Steam Play" updates.

I had tried to manually install new versions of Proton, for example when stuck using Proton 4.x and no option for Proton 5.x I'm pretty sure I searched and installed Proton 5.x in steam but it never was visible in the compatibility settings, until I discovered that removing Boxtron allowed steam to recognize the new versions available. This is with minor version updates as well as major versions, for example I was stuck at 4.11-9 for a long time on one system when other systems had updated to 4.11-13

I have seen this issue on 3 separate Ubuntu based installs. The workaround has been effective on each of them.

Thanks for all your hard work, not sure if this is a Boxtron issue or a Steam issue, maybe with custom compatibility tool Steam auto-disables updates? I am having a hard time finding any detailed documentation on how Steam/Steamplay/Proton are integrated but probably looking in the wrong areas. If you have a link to any good documentation I will gladly search through it. Let me know if there is any troubleshooting or investigation you would like me to try and perform.

**Comment by samualblair** [comment]
No longer appears to be an issue, most likely was a steam bug that has been resolved. I suspected it resolved about the time proton 5.0.6 was out but wasn't sure. Since then I have upgraded my systems to Ubuntu 20.04 (although I doubt that is related) and when proton 5.0.7 was released they all upgraded without any problem.
